src/components/pages/bc-new-wallet-link.ts

```diff
@@ -0,0 +1,105 @@
+import {customElement} from 'lit/decorators.js';
+import {BitcoinConnectElement} from '../BitcoinConnectElement';
+import {withTwind} from '../twind/withTwind';
+import {html} from 'lit';
+import {classes} from '../css/classes';
+import '../internal/bci-button';
+import {albyIcon} from '../icons/connectors/albyIcon';
+
+@customElement('bc-new-wallet-link')
+export class NewWalletPage extends withTwind()(BitcoinConnectElement) {
+  override render() {
+    return html`<div>
+      <bc-navbar
+        class="flex w-full"
+        heading="Get a bitcoin lightning wallet"
+      ></bc-navbar>
+
+      <div class="flex gap-4 w-full my-6 px-8">
+        <div class="font-sans text-sm w-full">
+          <div class="flex flex-row items-center space-x-4">
+            <div>
+              <div class="w-20 h-20">
+                <img
+                  width="64"
+                  ,
+                  height="64"
+                  ,
+                  src="https://cdn.builder.io/api/v1/image/assets/TEMP/7b138fd9-88ce-4ae9-8b00-a2c25a97345b?apiKey=daf9aaa5ccf8419a82d48f35c1c13cd8&"
+                  className="aspect-[0.69] object-contain object-center w-full overflow-hidden"
+                />
+              </div>
+            </div>
+            <p class="text-sm text-justify">
+              For a quick setup of bitcoin lightning wallet we recommend signing
+              up for the
+              <a
+                href="https://getalby.com/#alby-account"
+                target="_blank"
+                class="no-underline font-bold ${classes['text-brand-mixed']} "
+                style="cursor: pointer;"
+                >Alby Account</a
+              >, which you can pair with the
+              <a
+                href="https://getalby.com/#alby-extension"
+                target="_blank"
+                class="no-underline font-bold ${classes['text-brand-mixed']} "
+                style="cursor: pointer;"
+                >Alby Browser Extension</a
+              >.
+            </p>
+          </div>
+          <br />
+          <br />
+          <div class="flex flex-row items-center space-x-4">
+            <div>
+              <div class="w-20 h-20">
+                <img
+                  width="64"
+                  ,
+                  height="64"
+                  ,
+                  srcset="
```

Comment:
can you select and copy the images as svg format from figma into new files in `components/icons/new-wallet` (make a new folder there) to be consistent with our other icons?

In the code you can just do for example `${albyIcon}`